app/services/transaction_search/date_range_searcher.rb

```diff
@@ -3,7 +3,7 @@ module TransactionSearch
   class DateRangeSearcher < BaseSearcher
 
     include DateHelper
-    FIELDS = %w(ordered_at fulfilled_at journal_or_statement_date).freeze
+    FIELDS = %w(ordered_at fulfilled_at reconciled_at journal_or_statement_date).freeze
```

Comment:
Journaling/Statementing happens before reconciliation. Can we put `reconciled_at` at the end of the list so the dates are chronological.